src/cable/connection.cr

```diff
@@ -1,84 +1,54 @@
 require "uuid"
 
 module Cable
-  class Connection
+  abstract class Connection
```

Comment:
I think this makes sense. What do you think @fernandes?

Reply:
yes, totally, specially because this class should never get instantiated itself.. way better! 👍